test/e2e/case38_install_operator_test.go

```diff
@@ -597,79 +597,161 @@ var _ = Describe("Test installing an operator from OperatorPolicy", Ordered, fun
 						Kind:       "Deployment",
 						APIVersion: "apps/v1",
 					},
-					Compliant: "Compliant",
-					Reason:    "Deployment Available",
+					Compliant: "NonCompliant",
+					Reason:    "Resource not found but should exist",
 				}},
 				metav1.Condition{
 					Type:    "DeploymentCompliant",
 					Status:  metav1.ConditionTrue,
-					Reason:  "DeploymentsAvailable",
-					Message: "All operator Deployments have their minimum availability",
+					Reason:  "NoExistingDeployments",
+					Message: "No existing operator Deployments",
 				},
-				"All operator Deployments have their minimum availability",
+				"No existing operator Deployments",
 			)
 		})
 	})
-
-	Describe("Test health checks on OLM resources on OperatorPolicy with failed CSV", Ordered, func() {
+	Describe("Test status reporting for CatalogSource", Ordered, func() {
 		const (
-			opPolYAML = "../resources/case38_operator_install/operator-policy-no-group-csv-fail.yaml"
-			opPolName = "oppol-no-allnamespaces"
+			OpPlcYAML  = "../resources/case38_operator_install/operator-policy-with-group.yaml"
+			OpPlcName  = "oppol-with-group"
+			subName    = "project-quay"
+			catSrcName = "operatorhubio-catalog"
 		)
+
```

Comment:
The GitHub diff here makes me suspect there was a merge conflict and we've ended up losing a test scenario.

Reply:
I think I forgot to add in one of the tests from the CSV/deployments PR, I'll add that in now.